pkg/statistics/store_collection_test.go

```diff
@@ -72,7 +72,7 @@ func TestStoreStatistics(t *testing.T) {
 	re.Equal(0, stats.Unhealthy)
 	re.Equal(0, stats.Disconnect)
 	re.Equal(1, stats.Tombstone)
-	re.Equal(8, stats.LowSpace)
```

Comment:
Maybe use another test to cover the low space situation?

Reply:
Changed the capacity of one of the stores, now we can know that the `lowspace` is a specified store.